examples/pulseq_2d_radial_golden_angle.py

```diff
@@ -0,0 +1,164 @@
+# %% [markdown]
+# # Reconstruction of 2D golden angle radial data from pulseq sequence
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# %% [markdown]
+# If you want to run this notebook in binder you need to still install the MRpro package.
+# This only needs to be done once in a binder session. Open a terminal (File -> New -> Terminal) and run:
+# ```
+# pip install -e ".[notebook]"
+# ```
+# This will install the MRpro package. Any other required python packages should already be present in this
+# docker image.
+
+# %%
+# Imports
+import shutil
+import tempfile
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import requests
+import torch
+from mrpro.data import CsmData
+from mrpro.data import DcfData
+from mrpro.data import IData
+from mrpro.data import KData
+from mrpro.data.traj_calculators import KTrajectoryIsmrmrd
+from mrpro.data.traj_calculators import KTrajectoryPulseq
+from mrpro.data.traj_calculators import KTrajectoryRadial2D
+from mrpro.operators import FourierOp
+from mrpro.operators import SensitivityOp
+
+# %%
+# define zenodo records URL and create a temporary directory and h5-file
+zenodo_url = 'https://zenodo.org/records/10854057/files/'
+fname = 'pulseq_radial_2D_402spokes_golden_angle_with_traj.h5'
+data_folder = Path(tempfile.mkdtemp())
+data_file = tempfile.NamedTemporaryFile(dir=data_folder, mode='wb', delete=False, suffix='.h5')
+
+# %%
+# Download raw data using requests
+response = requests.get(zenodo_url + fname, timeout=30)
+data_file.write(response.content)
+
+# %% [markdown]
+# ### Image reconstruction using KTrajectoryIsmrmrd
+# This will use the trajectory that is stored in the ISMRMRD file.
+
+# %%
+# Read the raw data and the trajectory from ISMRMRD file
+kdata = KData.from_file(data_file.name, KTrajectoryIsmrmrd())
+
+# Calculate dcf using the trajectory
+dcf = DcfData.from_traj_voronoi(kdata.traj)
+
+# Reconstruct average image for coil map estimation
```

Comment:
I would change this to "Define Fourier operator and reconstruct coil images" (same for the other two)